decidim-templates/app/controllers/decidim/templates/admin/questionnaire_templates_controller.rb

```diff
@@ -90,6 +92,30 @@ def update
           end
         end
 
+        def edit_questions
+          @form = form(Decidim::Forms::Admin::QuestionsForm).from_model(questionnaire)
+
+          render template: edit_questions_template
+        end
+
+        def update_questions
+          params["published_at"] = Time.current if params.has_key? "save_and_publish"
+          @form = form(Decidim::Forms::Admin::QuestionsForm).from_params(params)
+          Decidim::Forms::Admin::UpdateQuestions.call(@form, questionnaire, current_user) do
+            on(:ok) do
+              # i18n-tasks-use t("decidim.forms.admin.questionnaires.questions_form.update.success")
+              flash[:notice] = I18n.t("update.success", scope: i18n_flashes_scope)
+              redirect_to edit_questionnaire_template_path(template)
+            end
+
+            on(:invalid) do
+              # i18n-tasks-use t("decidim.forms.admin.questionnaires.update.invalid")
+              flash.now[:alert] = I18n.t("update.invalid", scope: i18n_flashes_scope)
+              render template: edit_questions_template
+            end
+          end
+        end
+
```

Comment:
I see no difference between these methods and the ones in the "HasQuestionnaire" concern. We should just add the concern in this controller and only override whatever we need. This also applies for the methods "blank_question," "question_types", etc

Reply:
Done